sources/packages/backend/apps/test-db-seeding/src/seed-executors/seed-executor.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Test, TestingModule } from "@nestjs/testing";
+import { SeedExecutor } from "./seed-executor";
+import { DiscoveryService } from "@golevelup/nestjs-discovery";
+import { MetadataScanner } from "@nestjs/core";
+
+describe("SeedExecuter", () => {
+  let service: SeedExecutor;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [SeedExecutor, DiscoveryService, MetadataScanner],
+    }).compile();
+
+    service = module.get<SeedExecutor>(SeedExecutor);
+  });
+
+  it("should be defined", () => {
```

Comment:
This will result in "SeedExecutor should be defined" but your checking for the service in SeedExecutor to be defined.